pep-0705.rst

```diff
@@ -93,14 +93,22 @@ Finally, this allows bringing the benefits of ``TypedDict`` to other mapping typ
 Specification
 =============
 
-The ``TypedMapping`` type is a protocol behaving almost identically to ``TypedDict``, except:
+A ``TypedMapping`` type defines a protocol with the same methods as :class:`~collections.abc.Mapping`, but with value types determined per-key as with ``TypedDict``.
 
-* The runtime type of a ``TypedMapping`` object is not constrained to be a ``dict``
-* No mutator methods (``__setitem__``, ``__delitem__``, ``update``, etc.) will be generated
-* A class definition defines a ``TypedMapping`` protocol if and only if ``TypedMapping`` appears directly in its class bases
-* Subclasses can narrow field types, in the same manner as other protocols
+Notable similarities to ``TypedDict``:
```

Comment:
This still reads like a tutorial, not a specification. I would frame it more around `collections.abc.Mapping`: `TypedMapping` supports the same operations as `Mapping[str, object]`, except that the defined keys produce more precise types around methods like `get` and `__getitem__`.

Reply:
Is this different from my change to line 96?

Reply:
It's not! Sorry for missing that line in the diff.